internal/security/scram_client.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2013 Shopify
```

Comment:
So is this one-to-one copy? Might be worth adding some comments / links to where it came from.

Reply:
I changed by inspiring to it with a reference to the example.